packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -110,6 +134,12 @@ RawKeyEvent _toRawKeyEvent(dynamic message) {
         metaState: message['metaState'] ?? 0,
       );
       break;
+    case 'fuchsia':
+      data = new RawKeyEventDataFuchsia(
+        hidUsage: message['hidUsage'] ?? 0,
+        codePoint: message['codePoint'] ?? 0,
+      );
+      break;
     default:
       throw new FlutterError('Unknown keymap for key events: $keymap');
   }
```

Comment:
I've made this switch follow the pattern from the other switch, so you'll hit the default case and trigger an exception.